### DISCUSSION TITLE
Add radio stations to playlist

### DISCUSSION THREAD

**Discussion by chxia1024** [navidrome/navidrome#2613] [Ideas]
For example, I would like to be able to create a playlist that contains both local music and radio stations. 
When I play the playlist, it is possible to choose whether to play the local music or some radio station.

**Comment by metalheim** [comment]
This doesn't make sense *in my opinion*
A playlist is usallally used so that it progresses from one song to the next when a song has ended. 
A Radio station doesn't have an end and streams endlessly.

So when you have a mixed playlist with a radio station, it will eventually be stuck on the radio and never jump to the next song.
Also not compatible with subsonic

**Reply by chxia1024** [reply]
Em, you're right. 
I need a way to centrally manage these music sources. 
For example, if I'm listening to the radio and there's an ad, I want to be able to manually switch to a local song, then switch back to the radio when the ad is over.

Maybe this is not a playlist requirement, but rather a need for manual management of my favourite music resource. 
Do you have any suggestions for how to achieve this?

**Comment by akelge** [comment]
I would like to have this feature implemented. While I understand what @metalheim says, that makes sense under a technical point of view, I would love to have a list of favourite radios and be able to change quickly radio station, just pressing next/prev buttons, if the radio I am listening to is playing something I don't like. Like the preset stations on a car stereo, where you can press buttons from 1 to 6 (usually) and switch to one of your favourite station.

**Comment by Badsheep** [comment]
I'd love to have that feature too !
That would also allow me to synchronize this "Radios" Playlist with the playlist I use in mpd to play audio in local.

**Comment by potamus-zero** [comment]
I would also love this feature. 

I could have a playlist with all my radio stations and switch between them using the next/prev media buttons on my stereo's remote control. 

I could also make a playlist that ends with a radio station, for when you don't want to hear your tracks again, but don't want silence. Currently I do this by playing the radio station, telling my playlist to play next, opening the queue, dragging the radio station to the bottom, then selecting the first track from the playlist in the queue. It would save these steps. 

I can't use this workaround to do the first case - can't have two radio stations in the queue by telling a radio station to play next. So maybe a compromise would be a "Play Next" button for radio stations?
